src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphEntityDAO.scala

```diff
@@ -71,6 +71,7 @@ class GraphEntityDAO extends EntityDAO with GraphDAO {
       workspace.addEdge(entity.entityType, newVertex)
       newVertex
     })
+    entityVertex.setProperty("_vaultId", entity.vaultId)
```

Comment:
I assume it gets set to `None` (`null`?). There's nothing in the [TinkerPop documentation](http://www.tinkerpop.com/docs/javadocs/blueprints/2.1.0/com/tinkerpop/blueprints/Element.html#setProperty(java.lang.String, java.lang.Object)) that suggests it'll explode. I'd be more worried about how it'd interpret `Some(vaultId)`.